### DISCUSSION TITLE
What level of GPU is required to generate subtitles for a 10-minute video within 3 minutes

### DISCUSSION THREAD

**Discussion by kewer127** [openai/whisper#1981] [Q&A] [answered]
I use whisper to generate video subtitles. However, even with my decent Linux server (AMD Ryzen Threadripper PRO 5965WX s (48) @ 3.800GHz), it takes 30 minutes to generate the text using the default medium model with all CPU parallelly. I have no concept of GPUs. What level of GPU would I need to generate subtitles for a 10-minute video within 3 minutes?

**Comment by phineas-pta** [comment, marked as answer]
just buy a recent nvidia gpu that can fit the whisper model u using

or try `whisper.cpp`

**Comment by kewer127** [comment]
Thanks! `whisper.cpp` is way faster on my server! Whoa!
